### PR TITLE
Fixed Jumbled sentence

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das  <70086051+bishal7679@users.noreply.github.com>

### CODE REVIEW

**Review by cpanato — DISMISSED** [review]
thanks!
i think that is good

**Comment by cpanato** [comment]
@bishal7679 please sign the DCO

**Comment by bishal7679** [comment]
> @bishal7679 please sign the DCO

How can i do that?

**Comment by bishal7679** [comment]
> > > @bishal7679 please sign the DCO
> > 
> > 
> > How can i do that?
> 
> https://github.com/sigstore/docs/pull/52/checks?check_run_id=10352910051

Now its good to go?

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
My sense is that the wording here is intentional because OpenID Connect verifies that someone is who they say they are, not who they are (it can verify that an identity is associated with a given account). We perhaps should reword it to use the word "identity" instead.

**Comment by bishal7679** [comment]
> My sense is that the wording here is intentional because OpenID Connect verifies that someone is who they say they are, not who they are (it can verify that an identity is associated with a given account). We perhaps should reword it to use the word "identity" instead.

Then what sentence I should use here

**Comment by ltagliaferri** [comment]
@bishal7679 this may need to be a wider discussion. I am not available until after the New Year.

**Comment by haydentherapper** [comment]
Resolved in another PR
